Tools/LyTestTools/tests/unit/test_launcher_android.py

```diff
@@ -97,17 +97,17 @@ def test_GetPid_PidHasMultipleValues_ReturnPid(self, mock_output):
         assert mock_output.call_count == 2
         assert under_test == '12'
 
-    def test_GenerateMapCommand_HasMapArg_ReturnsMapCommand(self):
-        args_list = ['random_arg', '+map', 'map_name', 'another_arg']
+    def test_GenerateMapCommand_HasLoadLevelArg_ReturnsMapCommand(self):
```

Comment:
Want to update the names of these tests?

i.e. `test_GenerateLoadLevelCommand_HasArg_ReturnsLoadLevelCommand` and `test_GenerateLoadLevelCommand_NoArg_ReturnsEmptyString`

Reply:
Ah good catch, must have overlooked the extra "map" instances in the test names.